src/gui/PasswordGeneratorWidget.ui

```diff
@@ -59,6 +59,50 @@
          </property>
         </spacer>
        </item>
+       <item>
+        <widget class="QLabel" name="passwordLengthLabel">
+         <property name="sizePolicy">
+          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+           <horstretch>0</horstretch>
+           <verstretch>0</verstretch>
+          </sizepolicy>
+         </property>
+         <property name="minimumSize">
+          <size>
+           <width>70</width>
+           <height>0</height>
+          </size>
+         </property>
+         <property name="maximumSize">
+          <size>
+           <width>16777215</width>
+           <height>30</height>
+          </size>
+         </property>
+         <property name="text">
+          <string>passwordLength</string>
```

Comment:
Same here.

Reply:
I don't think so. This should get overwritten at runtime with the slot function. Or is there a case where that doesn't happen? When I opened up the password generator there were already characters in the password field and the slot for the character count label fired and changed the text on it.

Reply:
A string in the `.ui` file will get automatically translated ot `passwordLength`. Someone just opened an issue to Transifex and asked what this text is. The default string should be something that can be translated, or maybe an empty string if it's going to be filled at runtime?

Reply:
I actually thought it was just the string that appeared in the editor. I was basing it off the other labels in that row and they didn't have translations either:

```
        <widget class="QLabel" name="strengthLabel">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string comment="Password strength">strength</string> <----- here
         </property>
```

```
        <widget class="QLabel" name="entropyLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>entropy</string> <------- here
         </property>
```

Do we need translations for these as well? And does the translation tool generate entries for these? Or would I need to add them manually? I haven't worked too much with translations before so this is kind of new for me.

Reply:
No actions required